defines.php

```diff
@@ -134,7 +134,23 @@
  * Define for vipgoci_git_diffs_fetch()
  */
 
-define( 'VIPGOCI_GIT_DIFF_CALC_CHANGES',		array ('+' => 'additions', '-' => 'deletions') ); 
+define( 'VIPGOCI_GIT_DIFF_CALC_CHANGES',		array ('+' => 'additions', '-' => 'deletions') );
 define( 'VIPGOCI_GIT_DIFF_DATA_SOURCE_GIT_REPO',	'local-git-repo' );
 define( 'VIPGOCI_GIT_DIFF_DATA_SOURCE_GITHUB_API',	'github-api' );
 
+
+
+/**
+ * Define file number of lines limit
+ */
+define( 'VIPGOCI_SKIPPED_FILES',	        'skipped-files'	);
+
+define( 'VIPGOCI_VALIDATION_MAXIMUM_LINES_LIMIT',   15000   );
```

Comment:
Would be best if this was configurable via command-line. Suggest that the option is named `--skip-large-files-maxlines` or similar.